### ISSUE TITLE
Composer support

### ISSUE THREAD

**Issue by jeffw16** [CanastaWiki/Canasta#115]
As per the initial specs doc I wrote last year, we need to add Composer support eventually

**Comment by olsonjaredm** [comment]
Agreed. Is it `user-extensions` or `extensions` in the latest? I lost track.

One use case is if an admin wants specific versions of given extensions that they know are compatible with their setup/environment. This could include extensions that are bundled in Canasta, but at versions different than the bundle. For this reason it makes sense to put the files in `extensions` as these take precedent over `canasta-extensions`.

`composer update` is invoked now in the Dockerfile for some deps. Does adding Composer support entail adding a Composer script that is instead invoked after container start, or should it continue to be in the Dockerfile?

**Comment by yaronkoren** [comment]
I think this can be closed - there is now a standard way of downloading extensions that have a Composer dependency, which is to include "composer update" as part of "additional steps" for that extension/skin in the YAML file. It seems to work fine, and handles all known cases.
